### ISSUE TITLE
[PROD] Emergencies Overview - Disaster categorization

### ISSUE THREAD

**Issue by ypyelab** [IFRCGo/go-frontend#2118]
## Issue
The emergencies by default have a 'Yellow' categorization label. This is problematic, as it hinders the needed manual-work to select the crisis categorization related to an emergency. We need to be able to see what steps were not done.  
Also, the pdf scrapper should be able to identify most of those categorization levels (standard template).

Re: Florent message 
"We really need to be better at reflecting the categorization on GO.
Latest example is about DRC, classified as RED with a DREF submitted as RED and a category in GO that is yellow."

## Steps to reproduce
Example: This emergency had a yellow categorization level https://go.ifrc.org/emergencies/5623#reports, even if its associated DREF report https://adore.ifrc.org/Download.aspx?FileId=458168 was in a well-formatted manner (with reference to the 'red' categorization). 
We had to manually change it from yellow to red.

This is probably happening as the default categorization level for emergencies is 0 (yellow): https://github.com/IFRCGo/go-api/blob/develop/api/models.py#L557

## Expected behaviour
- The pdf scrapper should be successfully extracting the categorization level in most of the cases. 
- If that is not the case, an indicative token should be used (e.g. '--'), instead of a 'yellow' label (instead of using 0 by default)

## Impact
Anyone looking at emergency pages 

## Is there a workaround?
Users can contact IM for a manual update

**Comment by tovari** [comment]
Indeed, the default category is yellow and we can change the category manually. 
It is allocated during the emergency creation, when the first FR is submitted. This might be before the DREF request, so maybe a category update could be triggered by the DREF. What else could trigger a category update? Maybe Emergency Appeals?

**Comment by nanometrenat** [comment]
xref https://github.com/IFRCGo/go-web-app/issues/684
